src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -68,4 +53,30 @@ public void climberPosition() {
   public void periodic() {
     // This method will be called once per scheduler run
   }
+
+  public double getCurrentPosition(){
+    return climberMotor.getSelectedSensorPosition()/6380;
+  }
+  
+  public void setCurrentLimit(double Current){
+    TalonFXConfiguration config = new TalonFXConfiguration();
+    config.statorCurrLimit.currentLimit = Current;
+    climberMotor.configAllSettings(config); 
+  }
+
+  public void setPosition(){
+    climberMotor.setSelectedSensorPosition(0);
+  }
+
+  public double currentDrawed(){
+    return climberMotor.getStatorCurrent();
+  }
+
+  public static boolean getLeftTriggerActive() {
+    return (RobotContainer.m_xbox.getLeftTriggerAxis() > 0);
```

Comment:
Why does this method only return something from a RobotContainer reference?  There is nothing related to the climber subsystem in this method so it does not really belong to the ClimberSubsystem.  Any code that references it should just read from the trigger via RobotContainer directly.

The same goes for the getRightTriggerActive() method below.